pyvsystems/account.py

```diff
@@ -399,26 +396,26 @@ def check_tx(self, tx_id, confirmations=0):
             raise NetworkException("Cannot check transaction in offline mode.")
         utx_res = self.chain.unconfirmed_tx(tx_id)
         if "id" in utx_res:
-            self.logger.error("Transaction {} is pending in UTX pool.".format(tx_id))
+            # "Transaction {} is pending in UTX pool.".format(tx_id)
```

Comment:
`.format(tx_id)` is meaningless. Change comment to `The transaction is pending in UTX pool.`